aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/UnitTests/HomeControllerIndexPost.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc;
+using Moq;
+using TestingControllersSample.Controllers;
+using TestingControllersSample.Core.Interfaces;
+using TestingControllersSample.Core.Model;
+using TestingControllersSample.ViewModels;
+using Xunit;
+namespace TestingControllerSample.Tests.UnitTests
+{
```

Comment:
see https://github.com/aspnet/Docs/issues/1054 which we agreed to a few weeks ago (no horiz lines)